scripts/code_style.py

```diff
@@ -191,6 +209,11 @@ def main() -> int:
                         help='only check the specified files (default with non-option arguments)')
     parser.add_argument('operands', nargs='*', metavar='FILE',
                         help='files to check (files MUST be known to git, if none: check all)')
+    parser.add_argument('-g', '--git', action='store_true',
```

Comment:
Minor: please keep the `--` options in alphabetical order so that they're easy to browse.

Reply:
This will conflict with https://github.com/Mbed-TLS/mbedtls/pull/7822 (which I finally completed today because I happened to need it), but I don't mind rebasing mine.